accepted/0000-sbom-command.md

```diff
@@ -0,0 +1,493 @@
+
+```json
+{
+  "name": "cdx:npm:package:development",
+  "value": "true"
+}
+```
+
+Similarly, there are named properties defined for identifying things like "bundled", "private", and "extraneous" dependencies. Dependencies will be annotated with this properties as appropriate.
+
+The CycloneDX specification also provides [fields](https://cyclonedx.org/docs/1.4/json/#components) for capturing other package metadata like author, license, website, etc. Not all packages provide this information, but these fields will be populated when the information is available.
+
+#### SPDX
+
+```json
+{
+  "name": "debug",
+  "SPDXID": "SPDXRef-Package-debug-4.3.4",
+  "versionInfo": "4.3.4",
+  "downloadLocation": "https://registry.npmjs.org/debug/-/debug-4.3.4.tgz",
+  "filesAnalyzed": false,
+  "externalRefs": [
+    {
+      "referenceCategory": "PACKAGE-MANAGER",
+      "referernceType": "npm",
+      "referenceLocator": "debug@4.3.4"
+    },
+    {
+      "referenceCategory": "PACKAGE-MANAGER",
+      "referernceType": "purl",
+      "referenceLocator": "pkg:npm/debug@4.3.4"
+    }
+  ],
+  "checksums": [
+    {
+      "algorithm": "SHA512",
+      "checksumValue": "3d15851ee494dde0ed4093ef9cd63b..."
+    }
+  ]
+}
+```
+
+The example record shown above conforms to version 2.3 of the SPDX [Package](https://spdx.github.io/spdx-spec/v2.3/package-information/) specification.
+
+The <code>[downloadLocation](https://spdx.github.io/spdx-spec/v2.3/package-information/#77-package-download-location-field)</code> field will report the resolved location calculated by Arborist. This may point to a package registry, or a git URL for dependencies referenced directly from git. The top-level project will specify a value of <code>NOASSERTION</code> for the download location as this information is not available.
+
+All packages will specify a `false` value for the <code>[filesAnaylzed](https://spdx.github.io/spdx-spec/v2.3/package-information/#78-files-analyzed-field)</code> field to signal that no assertions are being made about the files contained within a package.
+
+The <code>[externalRefs](https://spdx.github.io/spdx-spec/v2.3/package-information/#721-external-reference-field)</code> field will contain two <code>[PACKAGE-MANAGER](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f3-package-manager)</code> references, one using the <code>[npm](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f32-npm)</code> reference type and the other using the <code>[purl](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f35-purl)</code> reference type.
+
+
```

Comment:
Could you add the missing the section "Prior Art"
see https://github.com/npm/rfcs/blob/main/accepted/0000-template.md#prior-art

> {{This section is optional if there are no actual prior examples in other tools}}

Actually, almost all features, switches, functionality, and implications described in the RFC are already implemented in the already mentioned tool: [@cyclonedex/cyclonedx-npm](https://www.npmjs.com/package/%40cyclonedx/cyclonedx-npm)
(which had a predecessor: [cyclonedx-node-module v3](https://github.com/CycloneDX/cyclonedx-node-module/tree/3.x))